shoulderdx_rotate:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: arm_j3
    joint_speed: 1
    motor:
        id: 3
        init: 512
        min: 212
        max: 2000

shoulderdx_open:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: arm_j4
    joint_speed: 1
    motor:
        id: 4
        init: 512
        min: 212
        max: 1500

elbowdx_open:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: arm_j5
    joint_speed: 1
    motor:
        id: 5
        init: 512
        min: 212
        max: 812

shouldersx_rotate:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: arm_j6
    joint_speed: 1
    motor:
        id: 6
        init: 512
        min: 212
        max: 2000

shouldersx_open:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: arm_j7
    joint_speed: 1
    motor:
        id: 7
        init: 512
        min: 212
        max: 812

elbowsx_open:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: arm_j8
    joint_speed: 1
    motor:
        id: 8
        init: 512
        min: 212
        max: 812

elbowdx_rotate:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: arm_j9
    joint_speed: 1
    motor:
        id: 9
        init: 512
        min: 212
        max: 812

elbowsx_rotate:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: arm_j10
    joint_speed: 1
    motor:
        id: 10
        init: 512
        min: 212
        max: 812


pan_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: pan_joint
    joint_speed: 0.20
    motor:
        id: 2
        init: 512
        min: 316
        max: 708

tilt_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController 
    joint_name: tilt_joint
    joint_speed: 0.20
    motor:
        id: 1
        init: 512
        min: 316
        max: 708 
